### PR TITLE
Alex's branch to improve GGNs

### PR DESCRIPTION
**Pull request by vict0rsch**
* [x] Create `new_X` files where `X` is an existing model class
* [x] Add tag embeddings
	* [x] Same nb of dims (`new_embedding_dim = embedding_dim - tag_embedding_dim`)
	* [ ] Extra dimensions

### CODE REVIEW

File: ocpmodels/common/flags.py [hunk 1]
```diff
@@ -127,7 +125,9 @@ def add_core_args(self):
             default="nccl",
             help="Backend for DDP",
         )
-        self.parser.add_argument("--local_rank", default=0, type=int, help="Local rank")
+        self.parser.add_argument(
+            "--local_rank", default=0, type=int, help="Local rank"
```
Comment (vict0rsch) [inline ocpmodels/common/flags.py:129]: Is this just `black` messing around?

File: .gitignore [hunk 2]
```diff
@@ -111,3 +111,16 @@ Local
...
+script3.sh
+script4.sh
+output-job.txt
+output-job-2.txt
+output-job-3.txt
+output-job-4.txt
+error-job.txt
+error-job-2.txt
+error-job-3.txt
+error-job-4.txt
```
Comment (vict0rsch) [inline .gitignore:126]:
```
script*.sh
output-job*.sh
error-job*.sh
```

File: ocpmodels/models/schnet_base.py [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        num_gaussians: int = 50,
+        cutoff: float = 10.0,
+        max_num_neighbors: int = 32,
+        readout: str = "add",
+        atomref: Optional[torch.Tensor] = None,
+        tag_hidden_channels: int = 32,
+    ):
+        super().__init__()
+
+        import ase
```
Comment (vict0rsch) [inline ocpmodels/models/schnet_base.py:78]: Why here?

File: ocpmodels/models/schnet_base.py [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+        self.register_buffer("initial_atomref", atomref)
+        self.atomref = None
+        if atomref is not None:
+            self.atomref = Embedding(100, 1)
+            self.atomref.weight.data.copy_(atomref)
+
+        self.reset_parameters()
+
+    def reset_parameters(self):
+        self.embedding.reset_parameters()
```
Comment (vict0rsch) [inline ocpmodels/models/schnet_base.py:127]: \+ `self.tag_embedding`

File: ocpmodels/trainers/base_trainer.py [hunk 5]
```diff
@@ -118,9 +121,10 @@ def __init__(
             commit_hash = None
 
         logger_name = logger if isinstance(logger, str) else logger["name"]
+        n = model.pop("name")
```
Comment (vict0rsch) [inline ocpmodels/trainers/base_trainer.py:124]: calling `n` `model_name` would be more explicit :)

File: get_results_all_val.py [hunk 6]
```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+import time
+
+from ocpmodels.common.flags import flags
+from ocpmodels.common.registry import registry
+from ocpmodels.common.utils import build_config, setup_imports
+
```
Comment (vict0rsch) [inline get_results_all_val.py:8]:
```
if __name__ == "__main__":
```

File: ocpmodels/models/schnet_base.py (hunk 3 above)
Comment (AlexDuvalinho) [reply ocpmodels/models/schnet_base.py:78]: I did not modify that -- it's how it is on PyG.

File: ocpmodels/models/schnet_base.py (hunk 4 above)
Comment (AlexDuvalinho) [reply ocpmodels/models/schnet_base.py:127]: Yes

**Pull request merged**